cmd/krel/cmd/release_notes.go

```diff
@@ -85,6 +89,30 @@ func init() {
 		"version tag for the notes",
 	)
 
+	releaseNotesCmd.PersistentFlags().StringVarP(
+		&releaseNotesOpts.sigReleaseFork,
+		"sigrelease-fork",
+		"",
+		"",
+		"user's fork of k/sig-release repo (eg user/reponame)",
```

Comment:
Be clear in the usage note that this flag is going to be used to create PRs.

Reply:
I've rewritten the help text. I hope it is clearer now.